### ISSUE TITLE
Import isn't working

### ISSUE THREAD

**Issue by suedunham** [giterlizzi/dokuwiki-plugin-datatables#57]
I'm trying to import into a new dokuwiki an export file created from another. This backup was created just days ago, so it's not old. I select the file, import everything into the root directory, and check the box at the top to select all the files. I've tried both values for Overwrite existing files. I then click the import button and the plugin goes back to the select a file page without having imported anything.

**Comment by giterlizzi** [comment]
Hi, it's a dokuwiki-plugin-advanced issue ?

**Comment by suedunham** [comment]
Yes. Shoot, how did I get over here in datatables? Sorry about that.
